corehq/apps/app_manager/views/apps.py

```diff
@@ -280,19 +285,29 @@ def _get_setting(setting_type, setting_id):
     context.update({
         'smart_lang_display_enabled': getattr(app, 'smart_lang_display', False)
     })
-    # Not used in APP_MANAGER_V2
-    context['is_app_view'] = True
 
     if app.get_doc_type() == 'LinkedApplication':
         context['upstream_url'] = _get_upstream_url(app, request.couch_user)
         try:
-            context['master_version'] = app.get_master_version()
+            master_versions_by_id = get_latest_released_app_versions_by_app_id(app.domain)
```

Comment:
what part(s) of this try/except block could raise `RemoteRequestError`? could this be changed to a try/catch/else?

Reply:
c0c3d253fed